src/svgs/glyphs/consult-history-glyph.svg

```diff
@@ -0,0 +1,15 @@
+<svg viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
i'd consider renaming this to clipboards-glyph since this doesn't have to be strictly used for consultation history